src/integration/pages/idxPage/persian_afghanistan/amp.test.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @service persian
+ * @pathname /persian/afghanistan
+ */
+
+import runAmpTests from '../ampTests';
+
+describe('AMP persian_afghanistan IDX page', runAmpTests);
```

Comment:
This is just me nitpicking to feel free to ignore - you don't need to have the underscore between persian and afghanistan - you could use the index URL or just the words
```suggestion
describe('AMP persian afghanistan IDX page', runAmpTests);
```
```suggestion
describe('AMP persian/afghanistan IDX page', runAmpTests);
```